.github/scripts/setup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+cd examples/ethereum
+nohup make run-rosetta > /dev/null 2>&1 &
+
+sleep 60
```

Comment:
will this always work? sleep for constant time?

Reply:
yes, for now. Spin up a rosetta service is fast.